packages/pages/src/components/address/types.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * The type definition for an address.
+ *
+ * @public
+ */
+export interface AddressType {
+  line1: string;
+  line2?: string;
+  line3?: string;
+  city: string;
+  region?: string;
+  postalCode: string;
+  countryCode: string;
+  sublocality?: string;
+  extraDescription?: string;
+  localizedRegionName?: string; // previously in derived data
+  localizedCountryName?: string; // previously in derived data
+  ","?: string;
```

Comment:
I added this because we wanted separators (commas) to be explicitly declared in the formatter. Maybe it should be part of a new extended interface though?

```
export AddressFormatter extends AddressType {
  ',': string;
}
```

Reply:
Yeah I don't think this should be part of the address type. 

Can we do
```
export type AddressLine = ((keyof AddressType) | ",")[];
```
instead?